app/models/concerns/resultable.rb

```diff
@@ -179,6 +179,17 @@ def missed_combined_round_cutoff?
     @sorted_solves_with_index ||= solve_times.each_with_index.reject { |s, _| s.skipped? }.sort.freeze
   end
 
+  def tied_with?(other_result)
+    false if other_result.nil?
+    if %w[a m].include?(result.format_id)
```

Comment:
`format`s have a `sort_by` column which can be even less technical debt than your current approach of hard-coding certain formats.